docs/tutorials/index.rst

```diff
@@ -1,8 +1,16 @@
 Tutorials
 =========
 
+Browsing the tutorials
+----------------------
+
 The tutorials included with WaterTAP are available in the "tutorials" directory
-on the WaterTAP GitHub Homepage: `<https://github.com/watertap-org/watertap/tree/main/tutorials/00-index.ipynb>`_.
+of the WaterTAP GitHub repository: `<https://github.com/watertap-org/watertap/tree/main/tutorials/>`_.
```

Comment:
Can this link to the 00-index notebook, or will that perpetuate the issue? I thought it'd be nice to be taken to the landing page which describes all available tutorials with links.

Reply:
That's a good question, I don't know if I have a definite opinion on it. I guess that linking to the directory should more or less always work (since it just uses the default GitHub file browser viewer), but at the same time you'd have to rely on the filenames instead of the the extended description in the intro/index. :man_shrugging: One thing to consider is also that the relative links in the intro/index won't work in the GitHub view (at least I think that's the case), while they work just fine in nbviewer. So maybe one possibility could be to just point to nbviewer as the default static browser, and leave the GitHub browser as a fallback.